### PR TITLE
Collections search auto suggestion styling fixes

### PR DESCRIPTION
**Pull request by HussainVAM**

### CODE REVIEW

File: src/components/blocks/search-form/_search-form.scss [hunk 1]
```diff
@@ -8,17 +8,10 @@
...
-    }
-
-    @include mixins.breakpoints-bpMinMedium {
-      padding-top: 10px;
-      padding-bottom: 16px;
-    }
-
+    margin:auto;
+    width:100%;
+    max-width: 720px;
```
Comment (methodog) [inline src/components/blocks/search-form/_search-form.scss:13]: very trivial formatting thing here, space after colons, but not worth fixing if it passed linting.

**Review by methodog — COMMENTED** [review]
Make sure each variant is tested! (i'm sure you have.)

Not convinced this is better than just adding the same form padding to the suggestions, and not having to decouple the two etc form styles, i.e. 
```
.b-search-form[suggesting] & {
     ...
     @include mixins.breakpoints-bpMinSmall {
        padding: 10px 12%;
      }
      @include mixins.breakpoints-bpMinLarge {
        padding-left: 20%;
        padding-right: 20%;
      }
```

File: src/components/blocks/search-form/_search-form.scss (hunk 1 above)
Comment (HussainVAM) [reply src/components/blocks/search-form/_search-form.scss:13]: Thats really strange - I don't have spaces after these colons in vscode.

Comment (methodog) [reply src/components/blocks/search-form/_search-form.scss:13]: sorry, i wasn't clear - there are meant to be spaces after colons throughout

Comment (HussainVAM) [reply src/components/blocks/search-form/_search-form.scss:13]: Ah I see!  I hadn't seen this being used across files but we can defo use that convention going forward.

Comment (methodog) [reply src/components/blocks/search-form/_search-form.scss:13]: more concerned that this fix for autosuggest will leave the form looking different to where it lands on EtC. this issue should never have gone through design, which always tends to introduce unplanned design changes. this was just something we had overlooked and needed to fix, without design change.

**Pull request merged**